pylua/network.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+
+import socket
+import tornado.httpserver
+import tornado.websocket
+import tornado.ioloop
+import tornado.web
+
+#websocket-client
+from websocket import create_connection
+
+class LuaWebSocketClient(object):
+    def __init__(self):
+        print '\nLuaWebSocketClient ctor\n'
+        self.websocket = None
+
+    def open(self, url):
+        self.websocket = create_connection(url)
+
+    def close(self):
+        self.websocket.close()
+
+    def write(self, message):
+        self.websocket.send(message)
+
+    def read(self):
+        return self.websocket.recv()
+
+class LuaSocketClient(object):
+    def __init__(self):
+        self.client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+    def connect(self, host, port):
+        self.client.connect((host, port))
+
+    def read(self):
+        return self.client.recv(4096)
+
+    def read_all(self):
+        # need to be reimplemented
+        return self.client.recv(4096)
+
+    def write(self, data):
+        self.client.sendall(data)
+
+    def close(self):
+        self.client.close()
+
+# only for testing
+class LuaSocketServer(object):
+    def __init__(self):
+        self.server = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+    def listen(self, port):
+        self.server.bind(('localhost', port))
+        self.server.listen(0)
+
+    def get_connection(self):
+        while True:
+            connection, address = self.server.accept()
+            buf = connection.recv(64) #pylint: disable=no-member
+            if len(buf) > 0:
+                print buf
+                print address
+                break
+
+# only for testing
+class LuaWebSocketServer(object): #pylint: disable=too-few-public-methods
```

Comment:
@OHerasym, are you sure that this warning is still relevant? Can we fix it instead of suppression?